tests/integration/fixtures.py

```diff
@@ -206,6 +212,25 @@ def sns_topic(sns_client):
     sns_client.delete_topic(TopicArn=topic_arn)
 
 
+@pytest.fixture
```

Comment:
nit: Wondering if we should move these very service-specific fixtures (i.e., those that will likely be used only in a single place and not shared across multiple files) into the `test_kms.py` test file itself (if possible).